src/app/components/content-creator-page/content-creator-page.component.html

```diff
@@ -68,16 +68,15 @@
                         </div>
                         <div class="col-1"></div>
                     </div>
-
+                    
+                    <!--Create a button type radio group to select content format-->
```

Comment:
Are these buttons still a radio group? Seems like they are checkboxes.

Reply:
That is correct, the format refactor was to allow for multiple formats to be selected during the search as opposed to only one.  Therefore, checkboxes instead of radio buttons.

Reply:
Will change the comment to reflect that.

Reply:
Sorry, nevermind, that is the content creator page, those are in fact still radio buttons.

Reply:
Content Creation and Request Creation are still radio (due to Formats being discrete.) but the searches have Checkbox functionality.